examples/reaction_prediction/rexgen_direct/README.md

```diff
@@ -149,6 +149,14 @@ New datasets should be processed such that each line corresponds to the SMILES f
 The reactants are placed before `>>` and the product is placed after `>>`. The reactants are separated by `.`. 
 In addition, atom mapping information is provided.
 
+**Notice**
+
+```
+The map ids in the rxn should be consecutive， or it will report [the molAtomMapNumber issue](https://github.com/awslabs/dgl-lifesci/issues/33).
```

Comment:
Let's use `atom mapping numbers` rather than `map ids` for consistency.

Reply:
yes, it will be more clear.